### PR TITLE
added a joystick notification for the apriltag command

### PR DESCRIPTION
**Pull request by ghost**
## Describe your changes

- [ ] Bug fix
- [ ] New feature

Explain what you did:
> ...

## Issue ticket (add number)

> https://github.com/FRC2706/2024-2706-Robot-Code/issues/...

## Checklist before requesting a review
- [ ] I have performed a self-review of my code.
- [ ] I have tested thoroughly.
- [ ] I have written documentation and explained about my changes to the best of my ability.

### CODE REVIEW

**Review by weizhang-2706 — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/PhotonSubsystem.java [hunk 1]
```diff
@@ -117,12 +120,13 @@ public Command getWaitForDataCommand(int tagid){
...
  * the command to run
  */
-  public Command getAprilTagCommand(PhotonPositions spacePositions){
+  public Command getAprilTagCommand(PhotonPositions spacePositions, CommandXboxController driverStick){
     Command swerveRequirementCommand = Commands.run(() -> {}, SwerveSubsystem.getInstance())
             .withName("AprilTagCommandSwerveRequirement");
     if (spacePositions.hasWaypoint){
       return Commands.sequence(
         getWaitForDataCommand(spacePositions.id),
+        new RumbleJoystick(driverStick, RumbleType.kBothRumble, 0.8, 0.2, false),
```
Comment (ErikCald) [inline src/main/java/frc/robot/subsystems/PhotonSubsystem.java]: `new Schedule(new RumbleJoystick(driverStick, RumbleType.kBothRumble, 0.8, 0.2, false))`

Comment (ErikCald) [reply src/main/java/frc/robot/subsystems/PhotonSubsystem.java]: Add for both sides of the if statement (waypoint and no waypoint)

**Review by ErikCald — DISMISSED** [review]

File: src/main/java/frc/robot/subsystems/PhotonSubsystem.java (hunk 1 above)
Comment (ghost) [reply src/main/java/frc/robot/subsystems/PhotonSubsystem.java]: change made

File: src/main/java/frc/robot/subsystems/PhotonSubsystem.java [hunk 2]
```diff
@@ -117,12 +120,13 @@ public Command getWaitForDataCommand(int tagid){
...
  * the command to run
  */
-  public Command getAprilTagCommand(PhotonPositions spacePositions){
+  public Command getAprilTagCommand(PhotonPositions spacePositions, CommandXboxController driverStick){
     Command swerveRequirementCommand = Commands.run(() -> {}, SwerveSubsystem.getInstance())
             .withName("AprilTagCommandSwerveRequirement");
     if (spacePositions.hasWaypoint){
       return Commands.sequence(
         getWaitForDataCommand(spacePositions.id),
+        new Schedule(new RumbleJoystick(driverStick, RumbleType.kBothRumble, 0.8, 0.2, false)),
```
Comment (ErikCald) [inline src/main/java/frc/robot/subsystems/PhotonSubsystem.java]: `ScheduleCommand` instead of `Schedule`

**Review by ErikCald — APPROVED** [review]
Looks gooooooooooooooooood to me !!!!1!!!

**Review by weizhang-2706 — APPROVED** [review]

**Pull request merged**